docs/add-ins/authenticate-a-user-with-an-sso-token.md

```diff
@@ -47,8 +47,10 @@ The add-in gets an SSO token with client-side script. For more information, see
 In most scenarios, there would be little point to obtaining the access token, if your add-in does not pass it on to a server-side and use it there. For details on what your server-side could and should do, see [Add server-side code](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#add-server-side-code).
 
 > [!IMPORTANT]
-> When using the SSO token as an identity in an *Outlook* add-in, we recommend that you also [use the Exchange identity token](authenticate-a-user-with-an-identity-token.md) as an alternate identity. Users of your add-in may use multiple clients, and some may not support providing an SSO token. By using the Exchange identity token as an alternate, you can avoid having to prompt these users for credentials multiple times. For more information, see[Scenario: Implement single sign-on to your service in an Outlook add-in](implement-sso-in-outlook-add-in.md).
+> When using the SSO token as an identity in an *Outlook* add-in, we recommend that you also [use the Exchange identity token](authenticate-a-user-with-an-identity-token.md) as an alternate identity. Users of your add-in may use multiple clients, and some may not support providing an SSO token. By using the Exchange identity token as an alternate, you can avoid having to prompt these users for credentials multiple times. For more information, see [Scenario: Implement single sign-on to your service in an Outlook add-in](implement-sso-in-outlook-add-in.md).
 
 ## Resources
 
-For a sample Outlook add-in that uses the SSO token to access the Microsoft Graph API, see [AttachmentsDemo Sample Add-in](https://github.com/OfficeDev/outlook-add-in-attachments-demo).
\ No newline at end of file
+- For a sample Outlook add-in that uses the SSO token to access the Microsoft Graph API, see [AttachmentsDemo Sample Add-in](https://github.com/OfficeDev/outlook-add-in-attachments-demo).
+- [SSO API reference](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#sso-api-reference)
```

Comment:
@kbrandl: I didn't link to the "Using SSO with an Outlook add-in" section because it pretty much pointed users to the current article (i.e., authenticate-a-user-with-an-sso-token.md). I'm happy to also include to top level sso-in-office-add-ins in the See also section but I didn't before because it's referred to in the body of the current article. I also felt that it's not obvious that the SSO reference would be in that article hence my reason for calling it out.

Reply:
Thanks for the explanation, that makes sense. (I didn't read the entire article, so I didn't have full context.) In that case, I think it's fine as is.